gax-java/gax-grpc/src/main/java/com/google/api/gax/grpc/InstantiatingGrpcChannelProvider.java

```diff
@@ -334,8 +330,15 @@ static boolean isOnComputeEngine() {
     return false;
   }
 
-  private boolean canUseDirectPathWithUniverseDomain() {
-    return endpoint.contains("googleapis.com");
+  // Universe Domain configuration is currently only supported in the GDU
+  @VisibleForTesting
+  boolean canUseDirectPathWithUniverseDomain() {
+    if (endpoint.contains(Credentials.GOOGLE_DEFAULT_UNIVERSE)) {
+      return true;
+    }
+    // This is only logged if DirectPath is enabled
+    LOG.log(Level.WARNING, "DirectPath will only work in the the googleapis.com Universe Domain");
```

Comment:
Kept this log statement as it only triggers if these three conditions above are true: https://github.com/googleapis/sdk-platform-java/blob/909bdf997dcaa23ec91da93121a1ed4a5bae9949/gax-java/gax-grpc/src/main/java/com/google/api/gax/grpc/InstantiatingGrpcChannelProvider.java#L372-L374

Might be beneficial since the user is has all the other DirectPath configurations correct except for Universe Domain.

Reply:
It could be beneficial, but this check is called on every channel resizing, which by default is every minute, hence could cause spam logs to customers. That's why the `logDirectPathMisconfigure` was moved to the constructor initially, so I think we can skip this logging for now.